sql/core/src/main/scala/org/apache/spark/sql/execution/ExistingRDD.scala

```diff
@@ -70,7 +70,20 @@ object RDDConversions {
 object ExternalRDD {
 
   def apply[T: Encoder](rdd: RDD[T], session: SparkSession): LogicalPlan = {
-    val externalRdd = ExternalRDD(CatalystSerde.generateObjAttr[T], rdd)(session)
+    val attr = {
+      val attr = CatalystSerde.generateObjAttr[T]
```

Comment:
As the deserializer is not resolved yet, there will be exception when accessing its `nullable`.